src/libs/actions/IOU.ts

```diff
@@ -1197,6 +1205,15 @@ function getUpdateMoneyRequestParams(
         });
     }
 
+    if (transactionThread) {
```

Comment:
Shouldn't we add the same if statement for the optimistic above? Looks like the optimistic data exists always, but failure has a condition.